platform/kube/inject/inject.go

```diff
@@ -132,12 +132,19 @@ const (
 )
 
 // InitImageName returns the fully qualified image name for the istio
-// init image given a docker hub and tag
-func InitImageName(hub, tag string) string { return hub + "/proxy_init:" + tag }
+// init image given a docker hub and tag and debug flag
+func InitImageName(hub string, tag string, _ bool) string {
+	return hub + "/proxy_init:" + tag
+}
 
 // ProxyImageName returns the fully qualified image name for the istio
-// proxy image given a docker hub and tag.
-func ProxyImageName(hub, tag string) string { return hub + "/proxy_debug:" + tag }
+// proxy image given a docker hub and tag and whether to use debug or not.
```

Comment:
s/proxy image given a docker hub and tag and whether to use debug or not./proxy image given a docker hub, tag and whether to use debug or not.

Reply:
thanks ! but sorry already merged, next pr/time; I think it's readable enough/I was minimizing the diff so not critical to fix/followup for "and" -> ","